cap-ci/jobs/minibroker-integration-tests.tmpl

```diff
@@ -0,0 +1,57 @@
+{{ define "jobs_minibroker-integration-tests" }}
+- name: minibroker-integration-tests-{{ .scheduler }}-{{ .backend }}-{{ .avail }}
+  serial_groups: [{{ .scheduler }}-{{ .backend }}-{{ .avail }}]
+  public: false
+  plan:
+  - get: catapult
+  - get: minibroker-integration-tests
+  - get: s3.minibroker
+  - get: mits-gh-release
+  - get: {{ .backend }}-pool.kube-hosts
+    {{- if ne .position 0 }}
+    passed:
+    - {{ index .jobs_enabled (sub .position 1) }}-{{ .scheduler }}-{{ .backend }}-{{ .avail }}
+    {{- end }}
+    trigger: true
+  - task: test-{{ .scheduler }}
+    {{- if has .workertags .backend }}
+    tags: [{{ index .workertags .backend }}]
+    {{- end }}
+    timeout: 1h30m
+    input_mapping:
+      pool.kube-hosts: {{ .backend }}-pool.kube-hosts
+    config:
+      platform: linux
+      image_resource:
+        type: registry-image
+        source:
+          repository: splatform/catapult
+      inputs:
+      - name: catapult
+      - name: minibroker-integration-tests
+      - name: pool.kube-hosts
+      - name: s3.minibroker
+      - name: mits-gh-release
+      params:
+        QUIET_OUTPUT: true
+        DOWNLOAD_CATAPULT_DEPS: false
+        ENABLE_EIRINI: {{ eq .scheduler "eirini" }}
+{{- if eq .avail "ha" }}
+        HA: true
+{{- end }}
```

Comment:
This sounds like it could be written this way:

```suggestion
        HA: {{ eq .avail "ha" }}
```

Reply:
And now that I suggested this, I see that the previous line already does this same pattern.
That makes me raise an eyebrow if the script checking for `HA` is actually doing `if [ -n "${NA}"]` instead of `if [[ "${NA}" == "true" ]]`.

Reply:
mmm when HA is set to false, I can confirm that there are no HA jobs eg: https://concourse.suse.dev/teams/main/pipelines/prabal-AKS-brains, only sa jobs are set to true here.

Unless it is a hard recommendation, I would like to leave it as it is, as this pattern is present in all scripts and will require thorough testing if changed. eg: https://github.com/SUSE/cap/blob/master/cap-ci/jobs/cf-acceptance-tests-brain.tmpl#L33-L35 
let me know :)

Reply:
Since you already asserted that when HA is set to false, I don't think it'd require "thorough testing". IMO, it's just a matter of keeping an eye on the first pipeline run to ensure it did the right thing.

Since it's most likely not going to get us in trouble, improving the code quality is something we should strive for!

Reply:
@f0rmiga I tested the suggestion. This is the difference between the 2:
In fist case, if HA is set to false in config, in SA mode there is no value of HA which is set
In second case (`HA: {{ eq .avail "ha" }}`), the HA value is being set in SA mode as well (to false). Which should be ok